shared/constants/types/wallets.tsx

```diff
@@ -275,6 +275,27 @@ export type _AirdropDetails = {
 }
 export type AirdropDetails = I.RecordOf<_AirdropDetails>
 
+export type TrustlineAssetID = string
+export const makeTrustlineAssetID = (issuerAccountID: string, assetCode: string): TrustlineAssetID =>
+  `${issuerAccountID}-${assetCode}`
+
+export type _TrustlineAsset = {
+  assetCode: string
+  issuerAccountID: string
+  issuerVerifiedDomain: string
+  trustedLimit: number // 0 if not trusted
```

Comment:
what does this mean? it's just treated as a boolean

Reply:
We have a number from the service, where user can specify a maximum amount to trust. It's not in current design -- we either trust it to maximum amount or don't trust it. But since the data is already there I thought I'd plumb it through anyway in case we want to display it somewhere in the future.